src/xcode/ENA/ENA/Source/Scenes/Events/Store/Model/TraceLocation.swift

```diff
@@ -103,5 +103,10 @@ struct TraceLocation {
 
 		return Date() < endDate
 	}
+	
+	var qrCodeURL: String {
+		let encodedByteRepresentation = byteRepresentation.base32EncodedString
+		return String(format: "https://e.coronawarn.app/c1/%@", encodedByteRepresentation).uppercased()
```

Comment:
Is this the right URL? Looks different to what I have in memory. Was there a change in the spec?

Reply:
It's as per tech specs:

# <SIGNED_TRACE_LOCATION_BASE32> = BIPEY33...
https://e.coronawarn.app/c1/BIPEY33...

Reply:
'https://e.coronawarn.app/c1/' won't match with the expected URLs from the scanner. If they are correct the other stories need a change request - otherwise it's wrong here, please check.

Reply:
Clarified it with @mlenkeit and the URL is correct.

Reply:
Create a Jira ticket https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5952